crypto/bn/bn_rsa_fips186_4.c

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2018, Oracle and/or its affiliates.  All rights reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <openssl/bn.h>
+#include "bn_lcl.h"
+#include "internal/bn_int.h"
+
+/*
+ * FIPS 186-4 Table B.1. "Min length of auxilary primes p1, p2, q1, q2".
+ */
+static int bn_rsa_fips186_4_aux_prime_min_size(int nbits)
+{
+    int min_bits;
+    switch (nbits) {
+#if 0
+    case 1024:
+        min_bits = 101;
+        break;
+#endif
+    case 2048:
+        min_bits = 141;
+        break;
+    case 3072:
+        min_bits = 171;
+        break;
+    default:
```

Comment:
Needs to add support for > 3072 ... 4096 should be possible.

Reply:
I was following the SP800-56Br1 doc. It has changed in the 56Br2 docs.

Reply:
Code has now been updated.

Reply:
> Code has now been updated.

Has it? I still don't see support for >3072

Reply:
Err No. Not sure how I managed to lose those changes. Redoing.

Reply:
Did I do something wrong?